### PR TITLE
Fixing database query that prevented json deletion

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Discovered in https://github.com/ooni/probe/issues/1271

### CODE REVIEW

File: ooniprobe/Model/Database/Measurement.m [hunk 1]
```diff
@@ -19,12 +19,12 @@ + (SRKResultSet*)notUploadedMeasurements {
 
 + (SRKResultSet*)selectWithReportId:(NSString*)report_id {
     return [[[Measurement query] where:
-             [NSString stringWithFormat:@"%@ AND report_id = %@", UPLOADED_QUERY, report_id]] fetch];
+             [NSString stringWithFormat:@"%@ AND report_id = '%@'", REPORT_QUERY, report_id]] fetch];
```
Comment (bassosimone) [inline ooniprobe/Model/Database/Measurement.m:22]: Is there a library we can use to construct queries instead of concatenating strings?

**Review by bassosimone — CHANGES_REQUESTED** [review]
I think this issue here should make us reflect that we need better tools to construct queries.

🤔

File: ooniprobe/Model/Database/Measurement.m (hunk 1 above)
Comment (lorenzoPrimi) [reply ooniprobe/Model/Database/Measurement.m:22]: Where takes a string, so you need to construct the string somehow. I prefer to use *define* strings like `UPLOADED_QUERY` so I don't have to copy it around

**Review by bassosimone — APPROVED** [review]
🐳 

(My previous request for changes was actually wrong!)

**Pull request merged**